frontend/web/components/IntegrationList.js

```diff
@@ -253,6 +363,8 @@ class IntegrationList extends Component {
               }
             : null
         }
+        githubId={githubId}
```

Comment:
Can we make these github props just a generic meta field, otherwise as we add others that need this kind of custom stuff these props will get out of control

Reply:
Good, added.